interfaces/Portalicious/src/app/pages/project/project-registrations/project-registrations.page.ts

```diff
@@ -68,6 +73,11 @@ export class ProjectRegistrationsPageComponent {
     ),
   );
 
+  projectReponse = injectQuery(
+    this.projectApiService.getProject(this.projectId),
+  );
+  project = computed(() => this.projectReponse.data());
```

Comment:
```suggestion
  project = injectQuery(
    this.projectApiService.getProject(this.projectId),
  );
```

I think in most places (if not everywhere) we would just reference this.project.data() directly